### ISSUE TITLE
Add contributing guide

### ISSUE THREAD

**Issue by joshuasing** [hemilabs/heminetwork#166]
### Confirmation

- [X] I have checked for similar issues.

### Suggested solution

I think it would be useful if we had a `CONTRIBUTING.md` file as a guide to contributing to this repository.
It could contain basic guidelines for contributing to help improve the quality of contributions, as well as steps to create a pull request and our processes.

This file could also contain a "maintainers guide" that includes information on general maintenance of this repository, and how  we handle things like releases.

**Comment by jcvernaleo** [comment]
@joshuasing we have https://github.com/hemilabs/.github/blob/main/CONTRIBUTING.md now.  Is that sufficient to close this issue?

**Comment by joshuasing** [comment]
> [@joshuasing](https://github.com/joshuasing) we have [hemilabs/.github@`main`/CONTRIBUTING.md](https://github.com/hemilabs/.github/blob/main/CONTRIBUTING.md) now. Is that sufficient to close this issue?

Yeah, that will work for now. Most of the information is not at all relevant to this repository though.
